examples/src/main/kotlin/example/AxiosSearch.kt

```diff
@@ -28,15 +27,48 @@ interface AxiosState : RState {
 //Per Hypnosphi advice, change to common js way.
 //you should need "npm install axios --save" in advance in your project folder
 @JsModule("axios")
-external fun axios(config: dynamic): dynamic
+external fun axios(config: AxiosConfigSettings): dynamic
+
+//add simple type for this example
+external interface AxiosConfigSettings {
+    var url: String
+    var timeout: Number
+}
 
 
 class AxiosSearch(props: AxiosProps) : RComponent<AxiosProps, AxiosState>(props) {
     override fun AxiosState.init(props: AxiosProps) {
-        zipCode = "" 
+        zipCode = ""
         zipResult = ZipResult("", "", "")
     }
 
+    private fun remoteSearchZip(zipCode: String) {
+        val config: AxiosConfigSettings = js {
```

Comment:
You can now use `jsObject`, which is type-safe

In fact, `js` is just a shortcut for `jsObject<dynamic>`

Reply:
Thanks for the great advice. Have change in code:)